src/datastore/postgres/PgDatastore.ts

```diff
@@ -152,15 +152,23 @@ export class PgDatastore implements Datastore {
     }
 
     public async getEventBySlackId(slackChannel: string, slackTs: string): Promise<EventEntry|null> {
-        return this.postgresDb.oneOrNone(
+        log.debug(`getEventBySlackId: ${slackChannel} ${slackTs}`);
+        const events = await this.postgresDb.manyOrNone(
             "SELECT * FROM events WHERE slackChannel = ${slackChannel} AND slackTs = ${slackTs}",
-            { slackChannel, slackTs }, e => e && {
+            { slackChannel, slackTs }
+        );
+        if (events[0]) {
+            const e = events[0];
+            return {
                 roomId: e.roomid,
                 eventId: e.eventid,
                 slackChannelId: slackChannel,
                 slackTs,
                 _extras: JSON.parse(e.extras),
-            });
+            };
+        } else {
+            return null;
+        }
```

Comment:
/me makes a cheesy sales pitch about `LIMIT`.

```
        return this.postgresDb.oneOrNone(
            "SELECT * FROM events WHERE slackChannel = ${slackChannel} AND slackTs = ${slackTs} LIMIT 1",
            { slackChannel, slackTs }, e => e && {
                roomId: e.roomid,
                eventId: e.eventid,
                slackChannelId: slackChannel,
                slackTs,
                _extras: JSON.parse(e.extras),
            });
```

Reply:
totally forgot that was a thing. Clever!